### ISSUE TITLE
Need help regarding DICOM Images preprocessing

### ISSUE THREAD

**Issue by DevidasKushnure** [harsh1795/Intern-GE-HealthCare#2]
Hi Harsh,
I am working on CHAOS 2019 DICOM CT Scan dataset for Liver Segmentation which has training dataset with labels(Ground Truth Images) and Testing dataset.
I want to know details about the preprocessing of the DICOM image database for training the deep neural network (CNN).
I need your help to understand detail preprocessing step wise so that I will be in a position to apply the steps to the DICOM image database.
Expecting a reply from your side. Your help will be much appreciated. Thanks in advance.
